### PR TITLE
Add Django app backbone

### PR DESCRIPTION
**Pull request by fenryxo**

### CODE REVIEW

**Comment by tpazderka** [comment]
I have turned on the travis integration, could you please do a new push to trigger it so we can test it?

**Comment by fenryxo** [comment]
It seems to work :-)

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+depends = clear-coverage
+passenv =
+    CI TRAVIS TRAVIS_*
+deps =
+    codecov
+    coverage
+extras =
+    tests
+commands =
+    coverage run --parallel --source=eidas_proxy_service --branch -m unittest {posargs:discover}
```
Comment (tpazderka) [inline tox.ini]: As a django app, this should probably use django admin to run the tests.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[mypy-django.*]
+ignore_missing_imports = True
+
+[pydocstyle]
+add_ignore = D105,D107
+# D105: Missing docstring in magic method
+#       They have a well-defined meaning, no need to duplicate their docstring.
+# D107: Missing docstring in __init__
+#       Arguments of __init__ method should be described in the class docstring.
```
Comment (tpazderka) [inline setup.cfg]: Missing newline.

File: eidas_proxy_service/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+"""Tests of eidas_proxy_service."""
```
Comment (tpazderka) [inline eidas_proxy_service/tests/__init__.py]: I think that this can go away now, as we have tested travis and codecov.

File: tox.ini (hunk 1 above)
Comment (fenryxo) [reply tox.ini]: Switched to django test.

File: setup.cfg (hunk 2 above)
Comment (fenryxo) [reply setup.cfg]: Added and editor config changed :-)

File: eidas_proxy_service/tests/__init__.py (hunk 3 above)
Comment (fenryxo) [reply eidas_proxy_service/tests/__init__.py]: Dummy test case removed.

**Pull request merged**